frontend/src/metabase/browse/components/MetricsTable.tsx

```diff
@@ -0,0 +1,286 @@
+  hideAtContainerBreakpoint: "sm",
+  containerName: itemsTableContainerName,
+};
+
+const collectionProps: ResponsiveProps = {
+  hideAtContainerBreakpoint: "xs",
+  containerName: itemsTableContainerName,
+};
+
+export function MetricsTable({
+  skeleton = false,
+  metrics = [],
+}: MetricsTableProps) {
+  const [sortingOptions, setSortingOptions] = useState<SortingOptions>(
+    DEFAULT_SORTING_OPTIONS,
+  );
+
+  const locale = useLocale();
+  const sortedMetrics = sortMetrics(metrics, sortingOptions, locale);
+
+  const handleSortingOptionsChange = skeleton ? undefined : setSortingOptions;
+
+  /** The name column has an explicitly set width. The remaining columns divide the remaining width. This is the percentage allocated to the collection column */
+  const collectionWidth = 38.5;
+  const descriptionWidth = 100 - collectionWidth;
+
+  return (
+    <Table aria-label={skeleton ? undefined : t`Table of metrics`}>
+      <colgroup>
+        {/* <col> for Name column */}
+        <ModelNameColumn {...nameProps} />
+
+        {/* <col> for Collection column */}
+        <TableColumn {...collectionProps} width={`${collectionWidth}%`} />
+
+        {/* <col> for Description column */}
+        <TableColumn {...descriptionProps} width={`${descriptionWidth}%`} />
+
+        <Columns.RightEdge.Col />
+      </colgroup>
+      <thead>
+        <tr>
+          <SortableColumnHeader
+            name="name"
+            sortingOptions={sortingOptions}
+            onSortingOptionsChange={handleSortingOptionsChange}
+            {...nameProps}
+            style={{ paddingInlineStart: ".625rem" }}
+            columnHeaderProps={{
+              style: { paddingInlineEnd: ".5rem" },
+            }}
+          >
+            {t`Name`}
+          </SortableColumnHeader>
+          <SortableColumnHeader
+            name="collection"
+            sortingOptions={sortingOptions}
+            onSortingOptionsChange={handleSortingOptionsChange}
+            {...collectionProps}
+            columnHeaderProps={{
+              style: {
+                paddingInline: ".5rem",
+              },
+            }}
+          >
+            {t`Collection`}
+          </SortableColumnHeader>
+          <SortableColumnHeader
+            name="description"
+            sortingOptions={sortingOptions}
+            onSortingOptionsChange={handleSortingOptionsChange}
+            {...descriptionProps}
+            columnHeaderProps={{
+              style: {
+                paddingInline: ".5rem",
+              },
+            }}
+          >
+            {t`Description`}
+          </SortableColumnHeader>
+          <Columns.RightEdge.Header />
+        </tr>
+      </thead>
+      <TBody>
+        {skeleton ? (
+          <Repeat times={7}>
+            <MetricRow />
+          </Repeat>
+        ) : (
+          sortedMetrics.map((metric: MetricResult) => (
+            <MetricRow metric={metric} key={`${metric.model}-${metric.id}`} />
```

Comment:
nit: `metric.model` is always the same, it does not need to be a part of the `key`